spec/object-spec.js

```diff
@@ -1,58 +1,66 @@
-'use strict';
-
-describe('Dog constructor', function(){
-  it("should create a new dog with three properties", function(){
-    var caldwell = new Dog("caldwell", "toy poodle", "4 months");
-    expect(caldwell.constructor == Dog).toEqual(true);
-    expect(caldwell.name).toEqual("caldwell");
-    expect(caldwell.age).toEqual("4 months");
-    expect(caldwell.breed).toEqual("toy poodle");
-  });
+const expect = require('expect');
+const jsdom = require('mocha-jsdom');
+const fs = require('fs');
+const path = require('path');
 
-});
+describe('objects', () => {
+  jsdom({
+    src: fs.readFileSync(path.resolve(__dirname, '..', 'objects.js'), 'utf-8')
+  })
+
+  describe('Dog constructor', function(){
+    it("should create a new dog with three properties", function(){
+      var caldwell = new Dog("caldwell", "toy poodle", "4 months");
+      expect(caldwell.constructor == Dog).toEqual(true);
```

Comment:
(Looks like that was leftover from the Jasmine specs, which are a little scary with their `==` instead of `===` (although in this case not a big deal). What do you think? Should we update these?)